bitbot/tarot.go

```diff
@@ -0,0 +1,117 @@
+package bitbot
+
+import (
+	"github.com/whyrusleeping/hellabot"
+	"math/rand"
+	"strings"
+	"fmt"
+)
+
+var TarotTrigger = NamedTrigger{
+	ID:   "Tarot",
+	Help: "Request tarot cards, default 1. Usage !tarot [num cards].",
+	Condition: func(irc *hbot.Bot, m *hbot.Message) bool {
+		return m.Command == "PRIVMSG" && strings.HasPrefix(m.Trailing, "!tarot")
+	},
+	Action: func(irc *hbot.Bot, m *hbot.Message) bool {
+		if (len(m.Content) > 7) {
+			resp := tarotCards[rand.Intn(len(magic8responses)-1)]
+			irc.Reply(m, resp)
+	} else {
+		msg := strings.TrimPrefix(m.Content, "!tarot ")
+        num := 1
+		fmt.Sscanf(msg, "%d", &num)
```

Comment:
If you're looking to cast an integer out of text input, try using `strconv.Atoi` instead
https://golang.org/pkg/strconv/#Atoi

Reply:
There is also no handling here for a non-castable string (ie, `!tarot potato`). Using `Atoi` and getting the `error` type back will help with that.